### DISCUSSION TITLE
Video File Limit More than 200MB?

### DISCUSSION THREAD

**Discussion by quantuan125** [absadiki/subsai#56] [Q&A] [answered]
(SOLVED) How to raise the file limit for the application? Is it possible to upload a larger video file than 200 MB?

 Thank you

**Comment by quantuan125** [comment, marked as answer]
Update: By changing the streamlight configuration file I was able to raise the limit of both file and message size

**Reply by absadiki** [reply]
yes you can increase the size limit from the streamlit configuration file, but I would suggest to use the command line interface for larger files. 
Thanks @quantuan125 for posting the answer.

**Reply by quantuan125** [reply]
Hi @abdeladim-s 

Thank you for the response. I have never used the command line interface before so how would I initiate that?

**Reply by absadiki** [reply]
You are welcome @quantuan125,
if you have never used the command line before then it may seem a little difficult to grasp. 

If you are on windows you need to open command line (or terminal) (the same as you did when you want to install `subsai`) but instead of launching the webui (with `subsai-webui`) you will need to do for example:
```shell
subsai /path/to/your/media/file.mp4 --model openai/whisper --model-configs '{"model_type": "small"}' --format srt
``` 
This will process the file (whatever its length) and generate the srt file in the same folder as the media file.

**Reply by quantuan125** [reply]
Hi @abdeladim-s 

I see, is there any advantageous of this method rather than changing the streamlit configuration? 

Also, I'm baby-stepping with python on visual code, is there a way to execute the code of your application there instead of doing in the window command prompt? If so, how can I able to do that (with which file from the subsai repo)? 

Thank you once again

**Reply by absadiki** [reply]
Hi @quantuan125,
* There are several advantages of the command line but besides it can process any file length, you can batch process many files at once (with the webui you have to process them manually one by one), you only need to feed a txt file with file paths and run it. this will save you a lot of time if you have so many files to transcribe. 

* Yes  you can, by executing the [webui.py file](https://github.com/abdeladim-s/subsai/blob/main/src/subsai/webui.py)

**Comment by quantuan125** [comment]
Fantastic answers. Thank you very much for resolving this part of my question! Cheers! @abdeladim-s

**Reply by absadiki** [reply]
You are welcome @quantuan125 :)
